submissions/mustbefail/oop/index.js

```diff
@@ -0,0 +1,86 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(name, gender, phrase, friends, legs) {
+    this.friends = friends ? friends : ['no friends'];
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.phrase = phrase;
+  }
+  saying() {
+    return this.phrase;
+  }
+
+  toString() {
+    return `Hello! I am <strong>${this.species}</strong>, my name is <strong>${
+      this.name
+    }</strong>, my gender is <strong>${this.gender}</strong>, I have <strong>${
+      this.hands
```

Comment:
This class knows nothing about child classes properties (and naturally shouldn't know). In theory you may have armless inhabitants and output will break.
Consider method overloading in child classes calling `super.parentClassMethod()` to method outcome.

Reply:
I hope I understood correctly what needs to be done

Reply:
Upd: Consider method overloading in child classes calling `super.parentClassMethod()` **to supplement child class method outcome**.

Reply:
done)